rpm/python-cryptography.spec

```diff
@@ -0,0 +1,68 @@
+%bcond_with tests
+
+%define srcname cryptography
+
+Name:           python-%{srcname}
+Version:        3.3.3
+Release:        0
+Summary:        PyCA's cryptography library
+
+License:        ASL 2.0 or BSD
+URL:            https://github.com/sailfishos/python-cryptography
+Source0:        %{name}-%{version}.tar.bz2
+
+BuildRequires:  openssl-devel
+BuildRequires:  gcc
```

Comment:
I wonder if we should have this or not... I mean if this causes extra builds or something for no good reason. It might not, I'm not sure. I know Fedora has started adding these to their packages but I don't remember if we had discussed requiring gcc in spec.